jarviscli/plugins/write_agenda.py

```diff
@@ -0,0 +1,30 @@
+from plugin import plugin
+import csv
+
+
+@plugin("write_agenda")
```

Comment:
```suggestion
@plugin("write agenda")
```

Reply:
Really?? I hate Travis jajaj.Thanks for the help!!